SlowCheetah.JDT.Tests/JsonTransformationTest.cs

```diff
@@ -0,0 +1,288 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah.JDT.Tests
+{
+    using System.IO;
+    using System.Text;
+    using Xunit;
+
+    /// <summary>
+    /// Test class for <see cref="JsonTransformation"/>
+    /// </summary>
+    public class JsonTransformationTest
+    {
+        private readonly JsonTransformationTestLogger logger = new JsonTransformationTestLogger();
```

Comment:
this is reused for each test, I think you need a way to clear out the contents of the logger at the beginning of a test.